### PR TITLE
MBS-6511: List of editors in the area

### PR DESCRIPTION
**Pull request by gentlecat**
I named the tab/page "Users" and not "Editors" to prevent confusion with "Edit" tab. Let me know if it shouldn't be like that.

### CODE REVIEW

**Comment by gentlecat** [comment]
I probably should add some tests for all this, right?

**Comment by mwiencek** [comment]
This looks really good, if you wanna try adding some tests (which would be nice), I'd create t/lib/t/MusicBrainz/Server/Controller/Area/Users.pm with the other files in that dir as a guide and just check that the page displays some editors...if you need to insert some test data you can do something like https://github.com/metabrainz/musicbrainz-server/blob/master/t/lib/t/MusicBrainz/Server/Controller/Series/Tags.pm#L14

**Comment by mwiencek** [comment]
> I named the tab/page "Users" and not "Editors" to prevent confusion with "Edit" tab. Let me know if it shouldn't be like that.

That seems fine, I think. We're not very consistent about it, or we mostly use "editors" in the context of editing, and "users" for the site in general.

**Comment by gentlecat** [comment]
Added some tests.

File: t/lib/t/MusicBrainz/Server/Controller/Area/Users.pm [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    $mech->content_contains('There are currently no users in this area');
+
+    $test->c->sql->do(q{
+        INSERT INTO editor (area, name, password, ha1)
+        VALUES (222, 'Editor 1', 'hunter2', '');
+    });
+
+    $mech->get_ok('/area/489ce91b-6658-3307-9877-795b68554c98/users');
+    html_ok($mech->content);
+    $mech->content_contains('There is currently 1 user in this area');
```
Comment (mwiencek) [inline t/lib/t/MusicBrainz/Server/Controller/Area/Users.pm]: How about also adding a content_contains for "Editor 1" and "Editor 2" below?

**Comment by mwiencek** [comment]
lgtm.

File: t/lib/t/MusicBrainz/Server/Controller/Area/Users.pm (hunk 1 above)
Comment (gentlecat) [inline t/lib/t/MusicBrainz/Server/Controller/Area/Users.pm]: Good idea. Done.

**Pull request merged**